test/Rln.t.sol

```diff
@@ -194,9 +192,58 @@ contract RlnTest is Test {
         rln.slash(idCommitment, to, zeroedProof);
         assertEq(rln.stakedAmounts(idCommitment), 0);
         assertEq(rln.members(idCommitment), 0);
+        assertEq(rln.memberExists(idCommitment), false);
 
         vm.prank(to);
         rln.withdraw();
         assertEq(rln.withdrawalBalance(to), 0);
     }
+
+    function test__root() public {
+        uint256[] memory idCommitments = new uint256[](10);
+        idCommitments[0] = 19143711682366759980911001457853255795836264632723844153354310748778748156460;
```

Comment:
no action point here. leaving this for reference, in case we want to generate other testvectors. isn't golang beautiful?

```go
package main

import (
	"math/big"

	log "github.com/sirupsen/logrus"
	"github.com/waku-org/go-zerokit-rln/rln"
)

func main() {

	rlnInstance, err := rln.NewRLN()
	if err != nil {
		log.Fatal(err)
	}

	leafsTestVector := []*big.Int{
		ParseBigInt("19143711682366759980911001457853255795836264632723844153354310748778748156460"),
		ParseBigInt("16984765328852711772291441487727981184905800779020079168989152080434188364678"),
		ParseBigInt("10972315136095845343447418815139813428649316683283020632475608655814722712541"),
		ParseBigInt("2709631781045191277266130708832884002577134582503944059038971337978087532997"),
		ParseBigInt("8255654132980945447086418574686169461187805238257784695584517016324877809505"),
		ParseBigInt("20291701150251695209910387548168084091751201746043024067531503187703236470983"),
		ParseBigInt("11817872986033932471261438074921403500882957864164537515599299873089437746577"),
		ParseBigInt("18475838919635792169148272767721284591038756730004222133003018558598315558783"),
		ParseBigInt("10612118277928165031660389522171737855229037400929675201853245490188277695983"),
		ParseBigInt("17318633845296358766427229711888486415250435256643711009388405482885762601797"),
	}

	for _, leaf := range leafsTestVector {
		rlnInstance.InsertMember(rln.BigIntToBytes32(leaf))
	}

	root, err := rlnInstance.GetMerkleRoot()
	if err != nil {
		log.Fatal(err)
	}

	log.Info("Merkle root:", rln.Bytes32ToBigInt(root))
	// 5210724218081541877101688952118136930297124697603087561558225712176057209122
}

func ParseBigInt(s string) *big.Int {
	x, ok := new(big.Int).SetString(s, 10)
	if !ok {
		log.Fatal("could not parse")
	}
	return x
}

```